package.json

```diff
@@ -23,8 +23,8 @@
     "balena-sdk": "^15.3.1",
     "bluebird": "^3.7.2",
     "compare-versions": "^3.6.0",
-    "docker-progress": "^4.0.3",
-    "dockerode": "^2.5.8",
+    "docker-progress": "^5.0.0",
+    "dockerode": "^3.3.0",
```

Comment:
The plan is to get the CLI to use `dockerode` v3 as well:
https://github.com/balena-io/balena-cli/pull/2259/files